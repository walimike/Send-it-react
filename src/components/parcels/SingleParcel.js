import React from "react";

const SingleParcel = props => {
  return (
    <tr>
      <td>{props.id}</td>
      <td>{props.description}</td>
      <td>{props.pickupLocation}</td>
      <td>{props.destination}</td>
      <td>{props.price}</td>
      <td>{props.status}</td>
    </tr>
  );
};

export default SingleParcel;