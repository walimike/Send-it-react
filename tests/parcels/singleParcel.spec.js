import React from 'react';
import { shallow } from 'enzyme';

import SingleParcel from '../../src/components/parcels/SingleParcel';

describe('single parcel', () => {
  const props = {
    redirectUser: jest.fn()
  }
  it('should render without crashing', () => {
    const wrapper = shallow(<SingleParcel />);
    expect(wrapper).toMatchSnapshot();
  });

});