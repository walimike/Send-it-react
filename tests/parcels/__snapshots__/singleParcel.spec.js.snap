// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single parcel should render without crashing 1`] = `ShallowWrapper {}`;
